src/test/metrics.test.ts

```diff
@@ -0,0 +1,323 @@
+import {suite} from 'uvu';
```

Comment:
Please include a copyright header at the top of this file, with a 2023 copyright date

Reply:
Done!

Reply:
Seems like the copyright header is still missing from this file.

Reply:
Oops, just pushed up the header to this file.